charts/openfunction/values.yaml

```diff
@@ -0,0 +1,95 @@
+config:
+  Example: |
+    knative-serving.namespace: "knative-serving"
+    knative-serving.config-features.name: "config-features"
+    # Configuration of the order of the plugins
+    plugins: |
+      pre:
+      - plugin1
+      - plugin2
+      post:
+      - plugin2
+      - plugin1
+    plugins.tracing: |
+      # Switch for tracing, default to false
+      enabled: true
```

Comment:
default should be false, not ture

Reply:
https://github.com/OpenFunction/OpenFunction/blob/main/config/bundle.yaml#L15292

Reply:
The original is not correct, correct it together